### DISCUSSION TITLE
Jamulus Usecase "Many to One"

### DISCUSSION THREAD

**Discussion by LouisWu13** [jamulussoftware/jamulus#2782] [General]
Hello, in the available online documentation for JAMULUS i mostly find the following use case: Each member of a band has his one JAMULUS Client and is playing from a remote site.

We have the following setup: Three band members are playing with one member, who uses JAMULUS. With this setting, we have the problem, that the singer's mic is recording as well the sound of the local guitar as the return signal from the jamulus remote host... so we run into a kind of infinite loopback.

Our setup is like this:

**Audio Interface:** TASCAM Model 12
MIC into Channel 3
Guitar into Channel 4
Keyboard in external amp, not connected to Interface/Mixer

Channel 1 of Mixer routed to channel 1 output of Jamulus
Channel 2 of mixer routed to channel 2 output of Jamulus

Jamulus input routed to channels 1 and 2 (USB Monitor) of mixer

The main question here is: how can we avoid this infinite loopback and provide the jamulus host with a clear sound signal and only get back his pure playing into the mixer.

My approach would be: mix the local signals (MIC, Guit1 and Key) in a DAW (Logic Pro) and send the sum to channel 1 out in Jamulus; send the Jamulus input to the DAW and mix it with the sum... - but will this really prevent loopback? And what does it do to the overall latency?

I would be happy, if anyone could point me to the right direction.

KR from the north german part of Europe Uwe

**Comment by chrisrimple** [comment]
First, you need to get all your incoming sounds down to 2 channels.  Mic, Guitar, and Keyboard all need to be mixed down to 2 channels.

Next, in Jamulus you need to set those 2 channels as the 2 inputs to Jamulus.

Finally, in Jamulus you need to set the outputs to 2 DIFFERENT channels.

For example, if you mix the instruments and mics in the Tascam to channels 1-2, those would be your Jamulus INPUTS.  But your Jamulus OUTPUTS need to be to different channels, like 7-8.  Then you plug your headphones into channels 7-8 to hear the sound coming from the Jamulus Server (including yourselves).

**Reply by LouisWu13** [reply]
I‘ll check that! Thx 🙏 and i‘ll give you a feedback soon!

**Comment by ann0see** [comment]
I don't fully understand your setup, but to me it seems comparable to my thread here: https://github.com/orgs/jamulussoftware/discussions/1800

If we come up with a solution, It's worth an article in the knowledge base, I'd say b
